### DISCUSSION TITLE
Search function

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#407] [Ideas] [answered]
I have gotten some signals recently that people are not finding what they are looking for through the search function. Based on their feedback I think the following should help:
- Search function also searches in titles and text (not only tags)
- Search function also searches partially correct matches. For example: searching for _chat_ will also give a hit on _chatbot_ (and preferably also vice versa).

**Comment by techsmyth** [comment, marked as answer]
Hi Flori, 

thanks - there is an epic pretty much at the top of the backlog to expand / refine the search: https://github.com/alkem-io/alkemio/issues/368

Good to look at some particular use cases. Feel free to add any addition details directly into the comments on the epic above.

Cheers, Neil
